Subject: [PATCH] Fix `all` target in Makefile
Author: Agathe Porte <agathe.porte@oss.qualcomm.com>

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,10 @@
 VIDEO_COMPILE_HOST = $(shell uname -n)
 VIDEO_GEN_PATH = $(VIDEO_ROOT)/driver/vidc/inc/video_generated_h
 
-all: modules
+SRC := $(shell pwd)
+
+all:
+	$(MAKE) -C $(KERNEL_SRC) M=$(SRC) modules $(KBUILD_OPTIONS)
 
 $(VIDEO_GEN_PATH): $(shell find . -type f \( -iname \*.c -o -iname \*.h -o -iname \*.mk \))
 	echo '#define VIDEO_COMPILE_TIME "$(VIDEO_COMPILE_TIME)"' > $(VIDEO_GEN_PATH)
